jstreegrid.js

```diff
@@ -170,7 +170,19 @@
 					sortAsc: true,
 					fixedHeader: s.fixedHeader || true,
 					width: s.width,
-					height: s.height
+					height: s.height,
+					gridcontextmenu : s.gridcontextmenu || function (grid,tree,node,val,col,t,target)
```

Comment:
Per the comment, if we save the whole function as a `defaultContextMenu`. Then:

````javascript
// in the above
gridcontextmenu : s.gridcontextmenu

// and after
if (gridcontextmenu === true) {
  gridcontextmenu = function(grid, tree,node,val,col,t,target) {...}
} else if (!!gridcontextmenu === false) {
  gridcontextmenu = false;
} 
````